.github/workflows/continuous_integration.yml

```
@@ -17,6 +17,14 @@ jobs:
         submodules: true   # Fetch and checkout submodules
         fetch-depth: 0     # Ensure the full history is fetched, useful when dealing with submodules
 
+    - name: 📦 Cache NuGet packages
+      uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)